okta_rules/okta_mfa_password_reset_audit.yml

```diff
@@ -0,0 +1,22 @@
+AnalysisType: scheduled_query
+QueryName: Okta Investigate MFA and Password resets
+Enabled: true
+Description: >
+  Investigate Password and MFA resets for the last 7 days 
+AthenaQuery: > 
+  SELECT p_event_time,actor.alternateId as actor_user,target[1].alternateId as target_user, eventType,client.ipAddress as ip_address
+  FROM panther_logs.okta_systemlog
+  WHERE eventType IN ('user.mfa.factor.reset_all', 'user.mfa.factor.deactivate', 'user.mfa.factor.suspend', 'user.account.reset_password', 'user.account.update_password')
```

Comment:
should "user.mfa.factor.update" be added to this query as well?

Reply:
Yes, Added